src/tests/api/config.test.ts

```diff
@@ -6,6 +6,17 @@ import { createDbConnection, closeDbConnection } from '../testdb';
 const adminSecret = 'Secrets are secretive';
```

Comment:
We should delete this. If it's being used we should import `env` above and then access `env.adminSecret`

Reply:
Looks like this pattern is in some other files as well, we should make sure that tests are using the imported `env`